.goreleaser.prerelease.yml

```diff
@@ -21,4 +21,30 @@ archives:
     - none*
 
 release:
-  prerelease: true
\ No newline at end of file
+  prerelease: true
+
+dockers:
+  - dockerfile: Dockerfile
+    image_templates:
+      - docker.pkg.github.com/chanzuckerberg/fogg/fogg:{{.ShortCommit}}
```

Comment:
I wonder if it is worth adding a comment as to why you are using the short commit instead of the version